build/fetch-browser.js

```diff
@@ -0,0 +1,499 @@
+(function (self) {
```

Comment:
Would you remove this browser build file please? It gets generated by npm when `npm publish` is run.

Reply:
Yes. I will do that. I had to add it as I was using this package from my fork in the browser.